pages/categories/categories.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: category
+title: Categories
+permalink: /categories
+---
+
+# Impact Categories
+
+The tech carbon standard categorises tech carbon emissions into three key sections:
+
+## Upstream Emissions (Category U)
+Emissions relating to the embodied carbon of hardware and the carbon emissions of the development and distribution of installed software used by the organisation [read more](/categories/upstream).
```

Comment:
Very picky, but would be inclined to put 'Read more' on a new line instead of within the sentences?

Reply:
Yup, done